lm_eval/utils.py

```diff
@@ -5,20 +5,18 @@
 import inspect
 import logging
 import os
-import pathlib
 import re
 import subprocess
 import sys
 from itertools import islice
-from typing import (
-    Any,
-    Callable,
-    List,
-)
+from pathlib import Path
+from typing import Any, Callable, List
 
 import yaml
 from jinja2 import BaseLoader, Environment, StrictUndefined
 
+from lm_eval.logging_utils import get_commit_from_path
```

Comment:
Does this cause a circular import between utils and logging_utils? I'd be okay with moving `get_git_commit_hash()` into logging_utils to avoid this, if it's okay with @baberabb !

Reply:
Yeah would be fine!

Reply:
I will move

Reply:
Moved `get_git_commit_hash` into `logging_utils`. Also moved add_env_info call from `__main__` to `evaluator` to be used also in the ways, which are tested by `test_evaluator`.

Reply:
Fantastic!